dsb-gui/src/test/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/FileTreeTableItemTest.java

```diff
@@ -43,14 +45,15 @@
 public class FileTreeTableItemTest {
 
     final private DateTimeUtils dateTimeUtils = new DateTimeUtils(DateTimeFormatter.ISO_LOCAL_DATE_TIME);
+    final private LoggingService loggingService = new LoggingServiceImpl();
```

Comment:
Do we need the real implementation for this test, or would a mock/stub be better for these tests?

Reply:
A mock would be better